gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Mar 07 09:42:03 CET 2017
+#Fri Jul 07 11:31:19 EEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
```

Comment:
Looks good, but if it's supported by buddy build, I'd bump to 4.0.1 directly so we can take advantage of the build cache. And if we bump to 4.0.1, I'd also add the [convenience method for adding google repo](https://docs.gradle.org/current/release-notes.html#convenience-method-for-adding-google-repository) in the build file.

Note: I just tested and everything seems fine with 4.0.1 locally.